docs/consume-packages/install-use-packages-visual-studio.md

```diff
@@ -53,7 +53,9 @@ To find and install a NuGet package with Visual Studio, follow these steps:
 
     - Select a package to see detailed package information.
     The details pane on the right appears and enables you to select a version to install.
-    ![Screenshot showing the NuGet Package Manager window with the Browse tab selected.](media/package-manager-browse-tab.png)
+    ![Screenshot showing the NuGet Package Manager window with the Browse tab selected.](media/package-manager-package-details.png)
+    You can see information about the owner(s), author(s), license, etc., on the Package Details tab and the package README file (if it is pprovided by the author) on the README tab.
```

Comment:
To help customers learn/use our vocabulary, maybe define it as "package metadata"?

Also, "by the author" makes sense in general, but I'm concerned about the confusion customers have between owner/author. Is there anything better to state here? Can we use "author" colloquially without it implying any special meaning? Maybe it's fine?

```suggestion
    You can see package metadata, information about the owner(s), author(s), license, etc., in the Package Details tab and the package README file (if it is provided by the package author) in the README tab.
```